health/notifications/alarm-notify.sh.in

```diff
@@ -2072,6 +2082,60 @@ EOF
 
   return 0
 }
+# -----------------------------------------------------------------------------
+# Opsgenie sender
+
+send_opsgenie() {
+  local payload httpcode oldv currv
+  [ "${SEND_OPSGENIE}" != "YES" ] && return 1
+
+  if [ -z "${OPSGENIE_API_KEY}" ] ; then
+    info "Can't send Opsgenie notification, because OPSGENIE_API_KEY is not defined"
+    return 1
+  fi
+
+  # We are sending null when values are nan to avoid errors while JSON message is parsed
+  [ "${old_value}" != "nan" ] && oldv="${old_value}" ||  oldv="null"
+  [ "${value}" != "nan" ] && currv="${value}" || currv="null"
```

Comment:
Please use tabs for indentation.

Reply:
@vlvkobal no, we use spaces, James and Austin agreed on format. We don't follow (at least me and Thiago, i don't say it is right/bad) _use code around format_, we add changes with new format.

But we need to reformat the file, yes.

Reply:
You are right. Then we need to reformat the file. Maybe in a separate PR.

Reply:
@thiagoftsm since you are using JB

<img width="732" alt="Screenshot 2020-10-28 at 13 08 28" src="https://user-images.githubusercontent.com/22274335/97422309-ae5d5e80-191e-11eb-9b4e-012b21251bf8.png">

With this config i have same format as James applied.

Reply:
I will make another PR to bring the unique pattern for the file, as @ilyam8 said.

Reply:
The general approach James and I had been using is reformatting the file as part of a PR that already changes it, with the formatting changes in a separate commit.

So for the moment, don’t do a PR just to fix the formatting, but please make a pass over the whole file next time you open a PR that would be modifying it anyway.